src/i18n/es/nav.ts

```diff
@@ -30,4 +30,5 @@ export default NavDictionary({
 	'guides/rss': 'RSS',
 	'guides/styling': 'Estilos & CSS',
 	'guides/troubleshooting': 'Solución de problemas',
+	'reference/directives-reference': 'Directivas de maquetado',
```

Comment:
Decime si te parece bien, me gustan ambas

```suggestion
	'reference/directives-reference': 'Directivas en maquetado',
```